weed mini restart panic

After stopping a "weed mini" run wirth CTRL-C , a 2nd run of "weed mini" cannot start the server stack. Instead I get a "panic: raft: failed to sync state dir: sync m9333: acess denied" error.

Below is the full log. How can I mitigate the situation other than removing the "m9333" directory completely before the next run?

**Version**: version 30GB 4.22 0b3cc8d121cf0e633c9081983171d3814eb16e2b windows amd64

❯ weed mini
I0430 15:03:03.955788 mini.go:793 Loaded 1 options from configuration file mini.options
I0430 15:03:04.006162 mini.go:662 Final port configuration - Master: 9333, Filer: 8888, Volume: 9340, S3: 8333, Iceberg: 8181, WebDAV: 7333, Admin: 23646
I0430 15:03:04.006162 mini.go:667 gRPC port configuration - Master: 19333, Filer: 18888, Volume: 19340, S3: 18333, Admin: 33646
I0430 15:03:04.006162 master.go:375 current: 10.0.11.144:9333 peers:none
I0430 15:03:04.006162 master.go:380 Running in single-master mode (peers=none), no quorum required
I0430 15:03:04.007198 file_util.go:28 Folder . Permission: -rwxrwxrwx
I0430 15:03:04.007198 mini.go:464 Optimal volume size: 1024MB (total disk capacity: 1953512MB, capacity/100 before rounding: 19535MB, rounded to nearest power of 2, clamped to [64,1024]MB)
I0430 15:03:04.007198 mini.go:985 Mini started with auto-calculated optimal volume size limit: 1024MB
I0430 15:03:04.007198 mini.go:1109 Master service starting...
I0430 15:03:04.007198 master.go:375 current: 10.0.11.144:9333 peers:none
I0430 15:03:04.007198 master.go:380 Running in single-master mode (peers=none), no quorum required
I0430 15:03:04.007198 master_server.go:153 Volume Size Limit is 1024 MB
I0430 15:03:04.007198 master.go:199 Start Seaweed Master 30GB 4.22 0b3cc8d121cf0e633c9081983171d3814eb16e2b at 0.0.0.0:9333
I0430 15:03:04.008739 raft_server.go:146 Starting RaftServer with 10.0.11.144:9333
I0430 15:03:04.008739 raft_server.go:78 Recovered TopologyId: bf6ae251-fe4e-491c-89c6-0b6ba328e37d
I0430 15:03:04.017678 raft_server.go:232 current cluster leader:
I0430 15:03:04.017678 master_server.go:253 [10.0.11.144:9333.19333]  - is the leader.
I0430 15:03:04.019235 master.go:264 Start Seaweed Master 30GB 4.22 0b3cc8d121cf0e633c9081983171d3814eb16e2b grpc server at 0.0.0.0:19333
I0430 15:03:04.019235 grpc_client_server.go:110 gRPC also listening on Unix socket /tmp/seaweedfs-master-grpc-19333.sock
panic: raft: failed to sync state dir: sync m9333: Zugriff verweigert

goroutine 159 [running]:
github.com/seaweedfs/raft.(*server).writeState(0x154b21acd7a0, 0x1, {0x154b22e0a060, 0x16})
        /go/pkg/mod/github.com/seaweedfs/raft@v1.1.7/server.go:1612 +0x267
github.com/seaweedfs/raft.(*server).candidateLoop(0x154b21acd7a0)
        /go/pkg/mod/github.com/seaweedfs/raft@v1.1.7/server.go:780 +0x26d
github.com/seaweedfs/raft.(*server).loop(0x154b21acd7a0)
        /go/pkg/mod/github.com/seaweedfs/raft@v1.1.7/server.go:622 +0x1b5
github.com/seaweedfs/raft.(*server).Start.func1()
        /go/pkg/mod/github.com/seaweedfs/raft@v1.1.7/server.go:470 +0x4c
created by github.com/seaweedfs/raft.(*server).Start in goroutine 170
        /go/pkg/mod/github.com/seaweedfs/raft@v1.1.7/server.go:468 +0x205

--- a/go.mod
+++ b/go.mod
@@ -67,7 +67,7 @@ require (
 	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475 // indirect
 	github.com/remyoudompheng/bigfft v0.0.0-20230129092748-24d4a6f8daec // indirect
 	github.com/seaweedfs/goexif v1.0.3
-	github.com/seaweedfs/raft v1.1.7
+	github.com/seaweedfs/raft v1.1.8
 	github.com/sirupsen/logrus v1.9.4-0.20230606125235-dd1b4c2e81af // indirect
 	github.com/spf13/afero v1.15.0 // indirect
 	github.com/spf13/cast v1.10.0 // indirect

